Makefile

```diff
@@ -0,0 +1,5 @@
+local: fullscreen.bs
+	bikeshed
+
+remote: fullscreen.bs
+	curl https://api.csswg.org/bikeshed/ -f -F file=@fullscreen.bs > fullscreen.html
```

Comment:
Note that this version doesn't give you warnings and such. But you do get to check the output.